Essentials/src/main/java/net/essentialsx/api/v2/services/package-info.java

```diff
@@ -0,0 +1,7 @@
+/**
+ * APIs that are registered with Bukkit's ServicesManager and can be accessed without touching Essentials internals.
```

Comment:
```suggestion
 * API services provided by EssentialsX. These are registered with Bukkit's ServicesManager, and can be accessed from there without getting an instance of EssentialsX first.
```
Should probably `{@link ServicesManager}` too